app/src/main/java/org/covidwatch/android/presentation/HomeFragment.kt

```diff
@@ -62,6 +62,8 @@ class HomeFragment : Fragment(), EasyPermissions.PermissionCallbacks {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        binding.swipeRefreshLayout.setColorSchemeColors(ContextCompat.getColor(requireContext(), R.color.colorPrimary))
```

Comment:
Shouldn't it use `colorPrimary` by default? Looks a bit hacky.

Reply:
It didn't, that's why I added it.

Reply:
Strange :)